Management/src/main/java/com/stack/jspdemo/controller/ProductController.java

```diff
@@ -0,0 +1,52 @@
+package com.stack.jspdemo.controller;
+
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import com.stack.jspdemo.model.Product;
+import com.stack.jspdemo.service.ProductService;
+
+@Controller
+public class ProductController {
+
+	@Autowired
```

Comment:
Is it necessary to use `@Autowired` annotation here?

Reply:
If we use constructor injection it is not necessary to use `@Autowired` as Spring will automatically inject those beans. 
For more information you can check this link: https://docs.spring.io/spring-boot/docs/current/reference/html/using-boot-spring-beans-and-dependency-injection.html

Reply:
done